README.md

```diff
@@ -85,23 +78,55 @@ You may also need to do another query that depends on the first one to get the r
 
 ```ruby
 resolve -> (obj, args, context) do
-  FindQuery.new(Product, args["id"]) do |product|
-    FindQuery.new(Image, product.image_id)
+  RecordLoader.for(Product).load(args["id"]).then do |product|
+    RecordLoader.for(Image).load(product.image_id)
   end
 end
 ```
 
-If the second query doesn't depend on the other one, then you can use GraphQL::Batch::QueryGroup, which allows each query in the group to be batched with other queries.
+If the second query doesn't depend on the first one, then you can use GraphQL::Batch.all, which allows each query in the group to be batched with other queries.
 
 ```ruby
 resolve -> (obj, args, context) do
-  smart_collection_query = CountQuery.new(SmartCollection, context.shop_id)
-  custom_collection_query = CountQuery.new(CustomCollection, context.shop_id)
-
-  QueryGroup.new([smart_collection_query, custom_collection_query]) do
-    smart_collection_query.result + custom_collection_query.result
+  GraphQL::Batch.all([
```

Comment:
This is a contrived example, so I'm not sure if it really represents real usage.  It is only used in 3 places in Shopify, and all those places are mapping an array to provide input, so that change would actually make it less convenient.